js2-mode.el

```diff
@@ -7821,6 +7821,9 @@ string is NAME.  Returns nil and keeps current token otherwise."
 (defun js2-get-prop-name-token ()
   (js2-get-token (and (>= js2-language-version 170) 'KEYWORD_IS_NAME)))
 
+(defun js2-peek-prop-name-token ()
+  (js2-peek-token (and (>= js2-language-version 170) 'KEYWORD_IS_NAME)))
```

Comment:
OK, I see why you did that, but I'd probably inline this function at its only call site.